sql/moz-fx-data-shared-prod/monitoring_derived/suggest_impression_rate_v1/metadata.yaml

```diff
@@ -0,0 +1,20 @@
+friendly_name: Suggest Impression Rate Stable
+description: |-
+  Used to visualize telemetry rates in Grafana as a cross check for
+  operational monitoring of the Merino service.
+owners:
+  - aplacitelli@mozilla.com
+labels:
+  authorized: true
```

Comment:
```suggestion
```

The `authorized` label is only for authorized views rather than queries populating a derived table.